src/Apps/Order2/Routes/Checkout/Components/Order2CollapsibleOrderSummary.tsx

```diff
@@ -0,0 +1,222 @@
+    artworkVersion?.date
+
+  const isRequiredOrderDataPresent =
+    orderData.buyerTotal?.display || orderData.itemsTotal?.display
+
+  // TODO: Handle missing/required data more intentionally.
+  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
+  useEffect(() => {
+    if (!isRequiredArtworkDataPresent) {
+      logger.error("Missing artwork data in Order2CollapsibleOrderSummary", {
+        artwork,
+        artworkVersion,
+      })
+    }
+    if (!isRequiredOrderDataPresent) {
+      logger.error("Missing order data in Order2CollapsibleOrderSummary", {
+        orderData,
+      })
+    }
+  }, [])
+
+  if (!isRequiredArtworkDataPresent) {
+    return null
+  }
+
+  const handleToggle = () => {
+    setIsExpanded(!isExpanded)
+  }
+
+  return (
+    <Box data-testid="OrderSummary" backgroundColor="mono0">
+      <Flex py={1} px={2} justifyContent="space-between">
+        <RouterLink flex={0} to={`/artwork/${artwork.slug}`} target="_blank">
+          <Image
+            mr={1}
+            src={artworkVersion.image.resized.url}
+            alt={artworkVersion.title}
+            width="40px"
+          />
+        </RouterLink>
+        <Box
+          overflow="hidden"
+          data-testid="OrderSummaryArtworkDetails"
+          style={{
+            whiteSpace: "nowrap",
+          }}
+          flex={1}
+          mr={2}
+        >
+          <Text
+            overflow="hidden"
+            style={{
+              textOverflow: "ellipsis",
+            }}
+            variant="xs"
+            color="mono100"
+          >
+            {artworkVersion.artistNames}
+          </Text>
+          <Text
+            overflow="hidden"
+            style={{
+              textOverflow: "ellipsis",
+            }}
+            variant="xs"
+            color="mono60"
+            textAlign="left"
+          >
+            {artworkVersion.title}, {artworkVersion.date}
+          </Text>
+        </Box>
+        <Clickable display="flex" onClick={handleToggle} flexShrink={0}>
+          <Text variant="xs" color="mono100" mr={0.5}>
+            {orderData.buyerTotal?.display || orderData.itemsTotal?.display}
+          </Text>
+
+          <ChevronDownIcon
+            height="18px"
+            mt="2px"
+            style={{
+              transition: "transform 0.3s ease-in", // Smooth flipping animation
+              transitionDelay: "0.15s", // Add a delay to the flip animation to help it line up with the height transition below
+              transform: isExpanded ? "scaleY(-1)" : "scaleY(1)", // Flip vertically when expanded
+            }}
+          />
+        </Clickable>
+      </Flex>
+      <Box
+        data-testid="OrderSummaryExpandedContent"
+        px={2}
+        overflow="hidden"
+        maxHeight={isExpanded ? "200px" : "0px"} // Set maxHeight to 0 when collapsed
+        style={{
+          transition: "max-height 0.3s ease-in", // Smooth transition for maxHeight
```

Comment:
Meta comment, but it would be good to worry about any and all animation at the very very end as a polish pass. Doing this stuff early just gets in the way